Cloudflare/network_gateway/tests/test_network.json

```diff
@@ -0,0 +1,55 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cloudflare Network Gateway",
+        "dialect_uuid": "d14567dd-56b1-42f8-aa64-fb65d4b0a4cf"
+      }
+    },
+    "message": "{\"AccountID\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"Action\":\"allowedOnNoRuleMatch\",\"Datetime\":\"2023-02-24T16:33:05Z\",\"DestinationIP\":\"104.244.42.193\",\"DestinationPort\":443,\"DeviceID\":\"\",\"DeviceName\":\"\",\"Email\":\"\",\"OverrideIP\":\"\",\"OverridePort\":0,\"PolicyID\":\"\",\"PolicyName\":\"\",\"SNI\":\"www.twitter.com\",\"SessionID\":\"1725de7a2d0000215517735400000001\",\"SourceIP\":\"15.188.186.81\",\"SourcePort\":34080,\"Transport\":\"tcp\",\"UserID\":\"\"}"
+  },
+  "expected": {
+    "message": "{\"AccountID\":\"1d1e650b3385b95db72bba7cfb1287e9\",\"Action\":\"allowedOnNoRuleMatch\",\"Datetime\":\"2023-02-24T16:33:05Z\",\"DestinationIP\":\"104.244.42.193\",\"DestinationPort\":443,\"DeviceID\":\"\",\"DeviceName\":\"\",\"Email\":\"\",\"OverrideIP\":\"\",\"OverridePort\":0,\"PolicyID\":\"\",\"PolicyName\":\"\",\"SNI\":\"www.twitter.com\",\"SessionID\":\"1725de7a2d0000215517735400000001\",\"SourceIP\":\"15.188.186.81\",\"SourcePort\":34080,\"Transport\":\"tcp\",\"UserID\":\"\"}",
+    "event": {
+      "category": [
+        "network"
+      ],
+      "dataset": "network_gateway",
+      "kind": "event",
+      "type": [
+        "allowed"
+      ]
+    },
+    "@timestamp": "2023-02-24T16:33:05Z",
+    "destination": {
+      "ip": "104.244.42.193",
+      "port": 443,
+      "address": "104.244.42.193"
+    },
+    "network": {
+      "transport": "tcp"
+    },
+    "source": {
+      "ip": "15.188.186.81",
+      "port": 34080,
+      "address": "15.188.186.81"
+    },
+    "cloudflare": {
+      "AccountID": "1d1e650b3385b95db72bba7cfb1287e9",
```

Comment:
I think this information may be duplicated in [`cloud.account.id`](https://www.elastic.co/guide/en/ecs/8.6/ecs-cloud.html#field-cloud-account-id)